src/devices/Lps25h/samples/Lps25h.Sample.cs

```diff
@@ -12,14 +14,23 @@ internal class Program
     {
         // I2C address on SenseHat board
         public const int I2cAddress = 0x5c;
-
+        
```

Comment:
remove extra spaces

Reply:
I don't understand what you're seeing. As much as I tried to remove these invisible changes, VS2019 can't see any changes. This is not important, I'm resolving this.